src/Controllers/Podcast/editPodcast.php

```diff
@@ -17,7 +18,7 @@
         $podcast = MyRadio_Podcast::create(
             $data['title'],
             $data['description'],
-            explode(' ', $data['tags']),
+            $data['tags'],
```

Comment:
i wonder whether it would be nicer if the (various) create functions took a list of tags, rather than an arbitrary string to be split...

Reply:
Hmm, maybe, personally given they only get used once, i'm not too fussed.